docs/cli/token.md

```diff
@@ -51,7 +51,7 @@ Bootstrap | `n/a`           | `n/a`
 
 If no token is provided when starting the first server in the cluster, one is created with a random password. The server token is always written to `/var/lib/rancher/k3s/server/token`, in secure format.
 
-The server token can be used to join both server and agent nodes to the cluster. It cannot be changed once the cluster has been created, and anyone with access to the server token essentially has full administrator access to the cluster. This token should be guarded carefully.
+The server token can be used to join both server and agent nodes to the cluster. Unless manually rotated with [`k3s token rotate`](#k3s-token-rotate), the server token does not change and anyone with access to the server token essentially has full administrator access to the cluster. This token should be guarded carefully.
```

Comment:
I don't think this belongs here. The fact that it can be rotated does not alter the fact that anyone who has access to it has full admin access.  If you want to call out that it can/should be rotated if compromised, do so in a separate paragraph or sentence.

Reply:
My intention was to change the sentence: `It cannot be changed once the cluster has been created`. This is not accurate because it surely can be changed with `k3s token rotate`. I can try to find some better wording

Reply:
Lets just take that out, and start the second sentence with "Anyone with access to the server token...". We can get into rotation later.